transform/torben_transform_samples.py

```diff
@@ -0,0 +1,239 @@
+_OUT_NODE_HEADERS = [
+    "id",
+    "name",
+    "category",
+]
+
+
+def load():
+    """
+    Load the source data and transform into data structures we can use to
+    easily generate a biolink formatted TSV
+    """
+    with open(_SOURCE_PATH) as fd:
+        reader = csv.reader(fd, delimiter='\t')
+        headers = next(reader)
+        # Output data
+        nodes = list()
+        edges = list()
+        # Validate headers
+        for header in headers:
+            if header not in _COLUMN_CONFIG:
+                raise RuntimeError(f"Header {header} not found in _COLUMN_CONFIG")
+        for idx, row in enumerate(reader):
+            # Validate row length
+            if len(row) != len(headers):
+                raise RuntimeError(f"Row {idx} has length {len(row)}, should be {len(headers)}")
+            # Iterate over columns
+            for (col_idx, col) in enumerate(row):
+                header = headers[col_idx]
+                config = _COLUMN_CONFIG[header]
+                col = col.strip()
+                if not config["node_category"] or _col_blank(col):
+                    # Skip this column
+                    continue
+                nodes.append({
+                    "id": config["prefix"] + ":" + col,
+                    "name": col,
+                    "category": config["node_category"],
+                })
+                # Create any edges to other columns
+                if config.get("edges_out"):
+                    for (col_name, edge_category) in config["edges_out"].items():
+                        if col_name not in headers:
+                            raise RuntimeError("Edge column '{col_name}' not in headers")
+                        if not _COLUMN_CONFIG[col_name]:
+                            raise RuntimeError("Edge column '{col_name}' has no config")
+                        obj_config = _COLUMN_CONFIG[col_name]
+                        obj_prefix = obj_config["prefix"]
+                        obj_idx = headers.index(col_name)
+                        obj_id = row[obj_idx].strip()
+                        if _col_blank(obj_id):
+                            continue
+                        edges.append({
+                            "subject": config["prefix"] + ":" + col,
+                            "object": obj_prefix + ":" + obj_id,
+                            "edge_label": edge_category,
+                            "relation": edge_category,
+                        })
+    return (nodes, edges)
+
+
+def _col_blank(col) -> bool:
+    """Is the column value null, unknown, or empty?"""
+    return not isinstance(col, str) or len(col.strip()) == 0 or "data not in IMG" in col
+
+
+def write(nodes, edges):
+    """
+    Write nodes and edges to TSV output files based on the _OUT_NODE_HEADERS
+    and _OUT_EDGE_HEADERS.
+    """
+    noext = os.path.splitext(_SOURCE_PATH)[0].replace('/source/', '/out/')
+    node_path = noext + '.biolink-nodes.tsv'
```

Comment:
can you align this file naming to:
IMGVR_extra_KGX_edges.tsv
IMGVR_extra_KGX_nodes.tsv

so subKG (Torben) dataset (sample) _KGX_edges.tsv etc

Also the format is KGX (and overlaps with many graph formats) and we are using some biolink terms to annotate our nodes/edges.